packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -466,8 +467,8 @@ class TimelineGrid extends CoreElement {
       // TODO(kenzie): Instead of calculating timestamp based on position, track
       // timestamp var and increment it by time interval represented by each
       // grid item. See comment on https://github.com/flutter/devtools/pull/325.
-      final timestamp = getTimestampForPosition(left + interval);
-      final gridItem = TimelineGridItem(left, interval, timestamp);
+      final timestampMs = getTimestampForPosition(left + interval);
```

Comment:
this all looks good. Maybe convert to Duration objects sooner? The fact that the variable names are better indicating a unit suggests we might want to store them in an object containing a unit.

Reply:
done.